processes/release_processes/README.MD

```diff
@@ -66,7 +66,8 @@ the release is published.
 
 #### Run the `publish_workflow.wdl` Workflow
 
-* Run the Cromwell instance using the local-backend.
+* Run the Cromwell instance using the Local backend. The publish workflow is quite resource intensive; it's a good idea to
+  shut down other resource intensive apps before launching it to avoid painfully slow or failed executions.
```

Comment:
➕  Also, I think I remember there being something about suggested memory allocation for docker suggesting that 4GB might be enough. It is not. I experienced an exceptionally long build that ultimately failed when using only 4GB. I went to 8GB which was fine, and I haven't been brave enough to try anything less. I recommend updating that bit of the documentation to remove the suggesting that 4GB might be enough.